scripts/collect_steps.py

```diff
@@ -16,15 +17,22 @@
     "--features=enable-benches debug-trace step-trace",
     "--",
     "--num-multi-bits",
-    "1",
+    "3",
 ]
 QUERY_SIZE = 10
 PER_USER_CAP = [1, 3]
 ATTRIBUTION_WINDOW = [0, 86400]
-BREAKDOWN_KEYS = [1, 33]
+BREAKDOWN_KEYS = [8, 33]
 SECURITY_MODEL = ["malicious", "semi-honest"]
 ROOT_STEP_PREFIX = "protocol/alloc::string::String::run-0"
-EXPECTED_HEADER_LINES_COUNT = 4
+
+DEPTH_DYNAMIC_STEPS = "ipa::protocol::attribution::InteractionPatternStep"
+BIT_DYNAMIC_STEPS = [
+    "ipa::protocol::attribution::aggregate_credit::Step::compute_equality_checks",
+    "ipa::protocol::attribution::aggregate_credit::Step::check_times_credit",
+]
```

Comment:
In the future, we want to do the same thing with a macro annotating dynamic steps and generate code. In order to do that, we need to be able to resolve the step's module path (or source file path) but that functionality is only available in nightly.